extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php

```diff
@@ -358,7 +358,7 @@ public function saveUserData($data) {
 		}
 
 		if (true === $changed) {
-			$this->user->setGlobalAttribute(self::USER_EVER_EDITED_MASTHEAD, true);
+			$this->user->setGlobalFlag(self::USER_EVER_EDITED_MASTHEAD, true);
```

Comment:
`self::USER_EVER_EDITED_MASTHEAD` is actually a flag, not an attribute. The [call to get this value](https://github.com/Wikia/app/blob/dev/extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php#L264-L266) uses $user->getGlobalFlag. This is just updating it to use the correct setter.